src/Symfony/Bundle/FrameworkBundle/Validator/ConstraintValidatorFactory.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Validator;
 
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use Psr\Container\ContainerInterface;
```

Comment:
about deprecating in this PR, see https://github.com/symfony/symfony/pull/21625#issuecomment-281973526
this needs to be resolved before acting to me.